src/runtime/database.h

```diff
@@ -60,6 +60,15 @@ struct Time {
   std::string as_string() const;
 };
 
+struct RunReflection {
+  int id;
+  Time time;
+  std::string cmdline;
+  RunReflection() = default;
```

Comment:
Do we need this constructor?

Reply:
We dont necessarily need it, but I used the constructor so its clearly understood what exactly I am querying: 
https://github.com/sifive/wake/pull/1667/files#diff-ce97135c77b7c227bcd205cb4d3449133611307c21539f37b8c422bde9780252R1591-R1593

Reply:
Ok thats fair. It we don't have any style convention here thats fine.